### ISSUE TITLE
xCon no longer working on today's TiVo 3.6 Release

### ISSUE THREAD

**Issue by bella92108** [n00neimp0rtant/xCon-Issues#515]
I've tried every current and past beta, and can't find one that'll patch today's TiVo 3.6 release's jailbreak detection.

Anyone else have any luck? Ideas?

**Comment by github22222** [comment]
I downgraded to previous version.

**Comment by bella92108** [comment]
I guess that's what I need to do until xCon is updated. Which version of xCon do you find works the best with the previous TiVo release?

**Comment by github22222** [comment]
I did not test but the but last beta was working very well. Sometimes the TiVo app would have to be restarted in order for the jail break to be hidden but it never failed me on iOS 8.4.

**Comment by bella92108** [comment]
Where can I download previous versions of TiVo.app? I tried my usual app to download it, but its not listed. Do you have a link to the last working version?

**Comment by github22222** [comment]
No. I followed advice from a Google search. Delete the TiVo app on the device and then plug it in to the computer. Sync with iTunes and you should see that the old TiVo app is ready to install. Do the install and you should have the old app back. I turned off automatic updates next but that should be done before.

**Comment by bella92108** [comment]
The issue is tivo 3.5.3 ipa has been updated in my iTunes too.  Can you copy tivo 3.5.3 from your iTunes and post it to a file share? 4share or mega or something

**Comment by github22222** [comment]
I will look into it but it will have to wait as I am not at home now. I have never done what you request so any guidance you might have would help me get it done.

**Comment by bella92108** [comment]
Got the version, but it's registered to someone else's iTunes account... and it's been forever, I don't remember how to modify an IPA so that it'll sync on my iTunes, or open on my iPad.

**Comment by DrD00der** [comment]
Damn.  Looks like downgrade time for me too.  Thankfully I still have the IPA in my iTunes but will have to turn off auto-update now for my apps!

**Comment by bella92108** [comment]
Wish xCon would update to work with the newest version. I don't see any of the other anti-jailbreak-detection tools working either.

**Comment by jkmbaum** [comment]
Anyone have the previous IPA file?

**Comment by alexgwallace** [comment]
It doesn't matter. I didn't update and streaming stopped working anyway. I
assume they're trying to force people to update.
